### PR TITLE
Add commands threat-grid-detonate-file and threat-grid-url-to-file

### PR DESCRIPTION
**Pull request by BenJoParadise**
This is a groundwork for ultimately closing issue https://github.com/demisto/etc/issues/9266
The command `threat-grid-detonate-file` does time out after 5 mintues (as of current design), a documentation issue noting this was opened: https://github.com/demisto/etc/issues/12046

(The issues above are intentionally mentioned in a way that will not close them when this PR is merged)  

Also resolves https://github.com/demisto/etc/issues/12091

### CODE REVIEW

**Comment by BenJoParadise** [comment]
Added, it will fail until the timeout is increased on the detonate-file command.
Please review.

File: TestPlaybooks/playbook-Threat_Grid_detonate_url_test.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+id: d99d5de0-72bd-4406-8001-a65bb50b65c3
+version: 1
```
Comment (anara123) [inline TestPlaybooks/playbook-Threat_Grid_detonate_url_test.yml]: version: -1

File: TestPlaybooks/playbook-Threat_Grid_detonate_url_test.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+id: d99d5de0-72bd-4406-8001-a65bb50b65c3
```
Comment (anara123) [inline TestPlaybooks/playbook-Threat_Grid_detonate_url_test.yml]: set the id to be something readable like threat_grid_detonate_url_test

**Comment by BenJoParadise** [comment]
Resolves https://github.com/demisto/etc/issues/12091

**Pull request merged**